### PR TITLE
Missing `mpmd.out` for NOAA AWS CSP

### LINKED ISSUE #2002: AWS mpmd failure
### What is wrong?

When running the global-workflow on NOAA CSP AWS, the workflow raises an exception regarding the absence of `mpmd.out` when running the GFS post-step.

### What should have happened?

The GFS post-step should not have raised an exception. It should behave how it does on the on-premise RDHPCS machines.

### What machines are impacted?

Cloud

### Steps to reproduce

Run a global-workflow application on NOAA CSP AWS with the in-line post-processing turned on.

### Additional information

N/A

### Do you have a proposed solution?

_No response_

### PR DESCRIPTION
**Pull request by HenryRWinterbottom**
# Description

This PR addresses issue #2002. The `env/AWSPW.env` file has been updated  with to now include the appropriate `mpmd.out` directives. Further, the `ush/fv3gfs_downstream_nems.sh` has been updated to allow PBS support. The PBS implementation may be considered a workaround for now and can be extended/modified if deemed necessary in the future. 

  Resolves #2002

# Type of change
- Bug fix (fixes something broken)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
This was tested on NOAA CSP AWS. The CI will test for the RDHPCS platforms.

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: ush/fv3gfs_downstream_nems.sh [hunk 1]
```diff
@@ -135,9 +135,16 @@ for (( nset=1 ; nset <= downset ; nset++ )); do
   fi
   err_chk
 
-  # We are in a loop over downset, save output from mpmd into nset specific output
-  cat mpmd.out  # so we capture output into the main logfile
-  mv mpmd.out "mpmd_${nset}.out"
+  # We are in a loop over downset, save output from mpmd into nset
+  # specific output; this if-block is necessary for the NOAA CSP AWS
+  # platform; AWS uses `mpiexec` (PBS) as it's parallel executable
```
Comment (aerorahul) [inline ush/fv3gfs_downstream_nems.sh:140]: WCOSS2 also uses PBS and is handled in the `elif [[ "${launcher:-}" =~ ^mpiexec.* ]]; then  # mpiexec` block.
As seen, the output of each thread is piped to `mpmd.${nm}.out`
How is AWS executing MPMD and where does its output go?  
The options above in `AWSPW.env` (`mpmd_opt="--multi-prog --output=mpmd.%j.%t.out"`) appear to be `SLURM` options.

File: env/AWSPW.env [hunk 2]
```diff
@@ -16,7 +16,7 @@ step=$1
 
 export npe_node_max=36
 export launcher="mpiexec.hydra"
-export mpmd_opt=""
+export mpmd_opt="--multi-prog --output=mpmd.%j.%t.out"
```
Comment (aerorahul) [inline env/AWSPW.env:19]: Are these the right options for `mpiexec.hydra`?  They appear to be slurm options

Comment (HenryRWinterbottom) [reply env/AWSPW.env:19]: @rajdpanda Can you please address @aerorahul's questions?

Comment (rajdpanda) [reply env/AWSPW.env:19]: that's correct! they are options for srun but not for mpiexec.hydra. Forgot to mention this to Henry. It can be taken out

Comment (HenryRWinterbottom) [reply env/AWSPW.env:19]: @rajdpanda So, the only change required is to `ush/fv3gfs_downstream_nems.sh`?

**Pull request closed without merging**